bigtop-deploy/puppet/hieradata/bigtop/cluster.yaml

```diff
@@ -69,15 +69,15 @@
 
 # Kerberos
 #hadoop::hadoop_security_authentication: "kerberos"
-#kerberos::site::domain: "do.main"
-#kerberos::site::realm: "DO.MAIN"
-#kerberos::site::kdc_server: "localhost"
-#kerberos::site::kdc_port: "88"
-#kerberos::site::admin_port: "749"
-#kerberos::site::keytab_export_dir: "/var/lib/bigtop_keytabs"
+#kerberos::krb_site::domain: "bigtop.apache.org"
+#kerberos::krb_site::realm: "BIGTOP.APACHE.ORG"
+#kerberos::krb_site::kdc_server: "%{hiera('bigtop::hadoop_head_node')}"
+#kerberos::krb_site::kdc_port: "88"
+#kerberos::krb_site::admin_port: "749"
+#kerberos::krb_site::keytab_export_dir: "/var/lib/bigtop_keytabs"
```

Comment:
May I know why the name is changed to from site to krb_site? I think this may break compatibility.

Reply:
The Hiera variables are not injected unless the name space of properties match the class name of Puppet classes. The relevant class name was changed from `site` to `krb_site` in 3386a9d2d6898be01852f1d340d8948161eacdcd. The commented out configs (as example) seem to be left unchanged.

Reply:
OK. That was me ;) Glad you catch this. Thanks!